maint/Makefile.PL.include

```diff
@@ -0,0 +1,29 @@
+BEGIN { -e 'Distar' or system("git clone git://git.shadowcat.co.uk/p5sagit/Distar.git") }
+use lib 'Distar/lib';
+use Distar 0.001;
+
+author 'Caelum: Rafael Kitover <rkitover@cpan.org>';
+
+manifest_include 't/backcompat/0.04006/lib' => qr/.*\.pm$/;
+manifest_include 't/bin' => qr/.*/;
+
+use FindBin;
+use lib "$FindBin::Bin/lib";
+require DBIx::Class::Schema::Loader::Optional::Dependencies;
+my $optdeps = 'DBIx::Class::Schema::Loader::Optional::Dependencies';
+my @req_groups = keys %{ $optdeps->req_group_list };
+my @rdbms_groups = grep { /rdbms/ } @req_groups;
+my @other_groups = grep { !/rdbms/ } @req_groups;
+our (%dev_requires, %runtime_suggests);
```

Comment:
I think this is a confusing way to communicate with the rest of the `Makefile.PL`.

Reply:
I don't disagree, but it was the only way I could think of that fit with how `Distar` works. Do you have an alternative?

Reply:
One option would be to return a hash from the script and do something like
```
my %dev_requires = -f 'META.yml' ? () : (do './maint/Makefile.PL.include' or die $@);
```
in `Makefile.PL`, but I'm not entirely sure I like that either.

Reply:
I've done this instead as a hash-ref, allowing for future expansion by adding to the returned list.